apps/examples/st_things/blink/sample_blink_things.c

```diff
@@ -80,6 +100,8 @@ int ess_process(void)
 		printf("[%s]=====================================================\n", TAG);
 		printf("[%s]                    Stack Started                    \n", TAG);
 		printf("[%s]=====================================================\n", TAG);
+	} else {
+		printf("[%s]Failed to start. Error code: %d\n", TAG, err_code);
```

Comment:
need to error turn is better.

Reply:
The caller function doesn't check return value whether is positive or negative and what else.